packages/frontend/core/src/components/explorer/context.ts

```
@@ -1,31 +1,50 @@
-  setView: Dispatch<SetStateAction<DocListItemView>>;
-  groups: Array<{ key: string; items: string[] }>;
-  collapsed: string[];
-  selectMode?: boolean;
-  selectedDocIds: string[];
-  prevCheckAnchorId?: string | null;
-  onToggleCollapse: (groupId: string) => void;
-  onToggleSelect: (docId: string) => void;
-  onSelect: Dispatch<SetStateAction<string[]>>;
-  setPrevCheckAnchorId: Dispatch<SetStateAction<string | null>>;
+export type DocExplorerContextType = {
+  view$: LiveData<DocListItemView>;
```

ESLint

[error] Finnish notation should not be used here.

(rxjs/finnish)